qualtran/bloqs/arithmetic/controlled_add_or_subtract_test.py

```diff
@@ -47,6 +47,7 @@ def test_examples(bloq_autotester):
     bloq_autotester(_ctrl_add_or_sub_signed_symb)
 
 
+@pytest.mark.slow
 @pytest.mark.parametrize("bitsize", [2, 4, pytest.param(5, marks=pytest.mark.slow)])
```

Comment:
Here only `5` is slow right?

On my machine:
```
2: 77ms
4: 1.271s
5: 5.428s
```

Reply:
You're right, I missed that `5` was already marked, so this one is good.

Reply:
I'm looking through the rest as well.